include/xplat/Starboard/SmartTypes.h

```diff
@@ -18,6 +18,11 @@
 
 #include "Starboard/TypeTraits.h"
 
+// IwFree symbol must already be declared when not building on Windows
+#ifndef _WIN32
```

Comment:
The issue was actually that IwMalloc.h drags in WinObjCRTExport.h, which is not xplat - I should probably avoid obfuscating this issue though.

---
In reply to: [72162785](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72162785) [](ancestors = 72162785)